support-frontend/assets/pages/digital-subscription-checkout/components/endSummary/endSummarySelector.ts

```diff
@@ -20,12 +21,14 @@ function mapStateToProps(state: CheckoutState): EndSummaryProps {
 		billingPeriod === 'Annual' ? billingPeriod : 'Monthly';
 	const digitalGiftBillingPeriod =
 		billingPeriod === 'Annual' ? billingPeriod : 'Quarterly';
+
 	return {
 		priceDescription: getBillingDescription(productPrice, digitalBillingPeriod),
 		promotion: getPromotion(productPrice) ?? '',
 		orderIsAGift: orderIsAGift ?? false,
 		digitalGiftBillingPeriod,
 		price: showPrice(productPrice),
+		isPatron: userIsPatron(productPrice.promotions),
```

Comment:
`EndSummary`'s props are derived from this `mapStateToProp` function, the `EndSummary` component expects to receive a prop `isPatron`, however this function wasn't returning `isPatron`, so `EndSummary` defaulted `isPatron` to `false` always. This was a bug, I've nosw updated this `mapStateToProps` to include `isPatron`.